stories/topics/proc-saas-set-up-private.adoc

```diff
@@ -0,0 +1,25 @@
+[id="proc-saas-set-up-private"]
+
+= Subscribing to an {SaaSonAWS} private offer
+A private offer has pricing and licensing terms specifically offered to your account.
+The offer has a set expiration date, if you don't accept the private offer by the expiration date, you are either automatically moved to the product's public offer or no longer subscribed to the product.
+
+.Procedure
+. A Red{nbsp}Hat seller issues you a purchase order and provides it to you by email.
+** Alternatively, for manual steps see link:https://docs.aws.amazon.com/marketplace/latest/buyerguide/buyer-private-offers.html#buyer-private-offers-subscribing[Viewing and subscribing to a private offer] page on the AWS Marketplace
+. Click the link in the email to accept the terms.
+. Click btn:[Pay now]. +
+This will direct you to {RHSSO} to create an account, or to log in if you already have an account.
+** If the AWS account you are subscribing from is not already associated with your Red{nbsp}Hat account it will redirect you to a page where you can do so. Once you have successfully associated your AWS account with your Red{nbsp}Hat account you will continue to step 5.
```

Comment:
Do you subscribe to something or from something?

it will direct -> it directs

When you have successfully

account, continue to Step 5.